lib/msg-format.c

```diff
@@ -45,6 +45,15 @@ msg_format_inject_parse_error(MsgFormatOptions *options, LogMessage *msg, const
 {
   GString *buf = scratch_buffers_alloc();
 
+  if ((options->flags & LP_PIGGYBACK_ERRORS) == 0)
```

Comment:
I think this should rather go to the call-site of `msg_format_inject_parse_error()`, because this condition basically makes this function a NOOP.

Reply:
> I think this should rather go to the call-site of `msg_format_inject_parse_error()`, because this condition basically makes this function a NOOP.

Fixed this up. Thanks for pointing this out.